### DISCUSSION TITLE
S3 Load error: [TIMEOUT]time out when read prefetch buffer

### DISCUSSION THREAD

**Discussion by holdending** [apache/doris#25144] [A - General / Q&A]
I created a S3 Load, and got an error message :
type:LOAD_RUN_FAIL; msg:errCode = 2, detailMessage = (****)[TIMEOUT]time out when read prefetch buffer

Has anyone get the same error and how to fix it, thanks!

**Comment by LemonLiTree** [comment]
which doris version？

**Reply by holdending** [reply]
thanks, my version is 2.0.  but now is fixed by add the prefix "http://"  to the s3 endpoint

**Comment by LemonLiTree** [comment]
https://github.com/apache/doris/pull/21153

**Reply by holdending** [reply]
thanks

**Comment by holdending** [comment]
done
